src/util/psbt/mod.rs

```diff
@@ -183,6 +185,103 @@ impl PartiallySignedTransaction {
 
         Ok(())
     }
+
+    // TODO: Work out attribution/license stuff, copied from `impl ComputeSighash for Legacy` in `bdk`.
```

Comment:
I think we can ask bdk for relicensing stuff here. They are super-friendly people and (I think) they would not mind doing it. But we should have attribution here.

Reply:
I've raised an issue on bdk repo to get input: https://github.com/bitcoindevkit/bdk/issues/586